src/components/Modals/AnnouncementModal.vue

```diff
@@ -30,8 +39,16 @@ export default {
     },
     announcement: {
       type: Object,
+      required: true,
     },
   },
+  created() {
+    this.USER = USER;
```

Comment:
Why is this necessary? where are these variables referenced?

Reply:
These are used in the :roles prop for AccessControl. Line 11 in this file.